assembly/src/parsers/mod.rs

```diff
@@ -475,6 +455,20 @@ fn check_div_by_zero(value: u64, op: &Token, param_idx: usize) -> Result<(), Par
     }
 }
 
+/// Checks that the u64 parameter value is a valid field element value and returns it as a field
+/// element.
+fn get_valid_felt(op: &Token, param_idx: usize, param: u64) -> Result<Felt, ParsingError> {
+    if param >= Felt::MODULUS {
+        return Err(ParsingError::invalid_param_with_reason(
+            op,
+            param_idx,
+            format!("parameter value must be smaller than {}", Felt::MODULUS).as_str(),
+        ));
+    }
+
+    Ok(Felt::new(param))
```

Comment:
As mentioned in one of my other comments, I don't think we need this function here. We could have a function which is something like `parse_felt()` in `field_ops` and it would be a just a wrapper around `parse_checked_param()`.